formats/detailed_guide/publisher/links.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "lead_organisations"
+  ],
+  "properties": {
+    "related_guides": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "alternative_format_provider_organisation": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "related_mainstream": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "additional_related_mainstream": {
```

Comment:
So something like this?

Before:

```json
// links.json:
"related_mainstream": {
  "$ref": "#/definitions/guid_list"
},
"additional_related_mainstream": {
  "$ref": "#/definitions/guid_list"
}

// And accompanying example:
"related_mainstream": [{
  "content_id": "aaa"
}],
"additional_related_mainstream": [{
  "content_id": "bbb"
}]
```

After:

```json
// links.json:
"related_mainstream": {
  "$ref": "#/definitions/guid_list"
}

// details.json
"related_mainstream_content": {
  "type": "string",
  "enum": ["main_first", "additional_first"]
}

// And accompanying example:
"related_mainstream": [{
  "content_id": "aaa"
}, {
  "content_id": "bbb"
}],
"details": {
  "related_mainstream_content": "main_first"
}
```

Could we just get rid of the details parameter altogether and establish that `main_first` is the only valid behaviour?

If we sometimes only need `additional_related_mainstream` content with no main `related_mainstream` content, we could potentially set `null` as the first item of the array, but that would require overriding the `links` type from `"$ref": "#/definitions/guid_list"` and potentially creating a new type of list, I think.